components/navbar.tsx

```diff
@@ -37,9 +37,7 @@ const Navbar = () => {
           </div>
         </div>
 
-        <nav
-          className={"px-2 pt-2 pb-4 sm:p-0 sm:flex " + (isOpen ? "block" : "hidden")}
-        >
+        <div className="px-2 pt-2 pb-4 sm:p-0 sm:block hidden">
```

Comment:
This `sm:block` should override `hidden` after the `sm:breakpoint` but it doesn't.